homeassistant/components/led_ble/util.py

```diff
@@ -0,0 +1,51 @@
+"""The yalexs_ble integration models."""
+from __future__ import annotations
+
+from homeassistant.components.bluetooth import (
+    BluetoothScanningMode,
+    BluetoothServiceInfoBleak,
+    async_discovered_service_info,
+    async_process_advertisements,
+)
+from homeassistant.components.bluetooth.match import ADDRESS, BluetoothCallbackMatcher
+from homeassistant.core import HomeAssistant, callback
+
+from .const import DEVICE_TIMEOUT
+
+
+@callback
+def async_find_existing_service_info(
+    hass: HomeAssistant, local_name: str, address: str
+) -> BluetoothServiceInfoBleak | None:
+    """Return the service info for the given local_name and address."""
+    for service_info in async_discovered_service_info(hass):
+        device = service_info.device
+        if device.address == address:
+            return service_info
+    return None
+
+
+async def async_get_service_info(
```

Comment:
This isn't used.

Reply:
Dropped here https://github.com/home-assistant/core/pull/81093